--- setup.py.orig	2016-07-15 10:51:46.477751200 +0200
+++ setup.py	2016-07-15 10:53:57.720781700 +0200
@@ -118,9 +118,13 @@
         # backwards compatibility.
         sdist.run(self)
 
+efilter_version = __version__
+if 'bdist_msi' in sys.argv or 'bdist_rpm' in sys.argv:
+ _, _, efilter_version = efilter_version.partition('!')
+
 
 setup(name="efilter",
-      version=__version__,
+      version=efilter_version,
       description="EFILTER query language",
       long_description=(
           "EFILTER is a general-purpose destructuring and search language "
