--- pefile.py.orig	2015-06-12 12:39:52.780304356 -0700
+++ pefile.py	2015-06-12 12:41:44.382797369 -0700
@@ -20,9 +20,9 @@
 the root of the distribution archive.
 """
 
-__revision__ = "$LastChangedRevision$"
+__revision__ = "139"
 __author__ = 'Ero Carrera'
-__version__ = '1.2.10-%d' % int( __revision__[21:-2] )
+__version__ = '1.2.10-%d' % int( __revision__ )
 __contact__ = 'ero.carrera@gmail.com'
 
 
